tasks/help.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const ClientKitTask = require('clientkit-task');
+const Logr = require('logr');
+class HelpTask extends ClientKitTask {
+
+  constructor(name, config, runner) {
+    config.log = new Logr({
+      type: 'cli',
+      renderOptions: {
+        cli: {
+          prefix: 'Clientkit Help Plugin:',
+          prefixColor: 'white',
+          lineColor: 'green'
+        }
+      }
+    });
+    super(name, config, runner);
+    this.calledOnce = false;
+  }
+
+  process(input, filename, processDone) {
+    if (this.calledOnce) {
+      return processDone();
```

Comment:
We should override execute and not process. Process won't get called because it doesn't have any files or items